release/lightgbm_tests/tpl_cpu_moderate_gce.yaml

```diff
@@ -0,0 +1,17 @@
+cloud_id: {{env["ANYSCALE_CLOUD_ID"]}}
+region: us-west1
+allowed_azs:
+    - us-west1-b
```

Comment:
I think us-west1-c has more machine if we don't need gpu, cc: @hanming-lu if it's not correct

Reply:
yes. Note that `us-west1-c` does not have any GPU. So pick it with care :)

Reply:
With `us-west1-c` I ran into the internal server error, with `-b` I didn't.

Since these are relatively small machines (and some use GPUs), can we stick with -b?